app/src/main/java/com/google/android/soundchecker/utils/ui/Waveform.kt

```diff
@@ -0,0 +1,72 @@
+package com.google.android.soundchecker.utils.ui
+
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment.Companion.Center
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.*
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun Waveform(
```

Comment:
The word "Waveform" refers to a periodic audio signal.
So I would expect this class to generate a waveform.
Since it displays a waveform please indicate that by including something graphical in the name like "Draw" or "Display" or "View" or ???